src/routes/sprungbrett/containers/SprungbrettOfferPage.js

```diff
@@ -33,6 +33,9 @@ export class SprungbrettOfferPage extends React.Component<PropsType> {
         <List noItemsMessage={t('noOffersAvailable')}
               renderItem={this.renderSprungbrettListItem}
               items={sprungbrettJobs} />
+        <a target='_blank' rel='noopener noreferrer' href='https://www.sprungbrett-intowork.de'>
```

Comment:
I accidentally forgot to mention in the issue that the image should be centered horizontally. Ups, sorry!

This also fixes problems with RTL and LTR layouts. Centered is the same in both directions :)

Reply:
Fixed this.